docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
+# Volume migration use S3 API
+
+## Overview
+
+This article contains instructions for migration an OpenStack Cinder volume between two OpenStack environments (specifically, between the MOC Kaizen cluster and the NERC OpenStack cluster, but the instructions will work for any two OpenStack environments that have the necessary services enabled).
```

Comment:
typo: ...instructions for *migration an* OpenStack Cinder Volume...
not sure what you mean here, "migration of an openstack cinder volume" or "migrating an openstack cinder volume".